ecommerce_extensions/tenant/extra_scripts.py

```diff
@@ -0,0 +1,94 @@
+"""
+This function gets called during every request by the
+context processor to return all the custom scripts for a specific path.
+"""
+import logging
+import re
+
+import six
+
+#from eox_theming.configuration import ThemingConfiguration
+
+logger = logging.getLogger(__name__)
+
+common_attributes = {'type': {'options': ['inline', 'external']},
+                     'media_type': {'options': ['module', 'text/javascript'], 'default': 1}}
+
+
+def validate_script_attributes(values):
+    """
+    Validate common attributes for external and inline scripts.
+    """
+    script_attributes = {}
+
+    # Validate common attributes for external and inline scripts
+    for key, attr_values in common_attributes.items():
+
+        value = values.get(key)
+
+        try:
+            if value is not None:
+                attr_values['options'].index(value.lower())
+
+                script_attributes[key] = value.lower()
+            else:
+                default = attr_values['default']
+
+                script_attributes[key] = attr_values['options'][default]
+
+        except Exception:  # pylint: disable=broad-except
```

Comment:
This block could raise KeyError or indexError you don't need to catch everything